src/es-router.js

```diff
@@ -16,11 +17,13 @@ class EsRouter {
     this.useHash = useHash;
     this.routes = routes;
     this.base = base;
-    this.notStrictRouting = notStrictRouting;
+    this.strictRouting = strictRouting;
     this.queryParams = this.getParamsFromUrl();
 
-    if (base && base[base.length - 1] === '/') {
+    if (base && base[base.length - 1] === '/' && base !== '/') {
```

Comment:
You would think so, but it wasn't working for me until I added that. I'll check again and update the pr if it's not needed